src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilter.java

```diff
@@ -407,6 +407,45 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example for trailing comments check to suppress specific trailing comment:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   int a; // SUPPRESS CHECKSTYLE, OK
+ *   int b; // NOPMD, OK
+ *   int c; // NOSONAR, OK
```

Comment:
These 2 words are weird and are requiring to be in the white list words when they aren't words. Can we change these words to be more english?

Reply:
These words are used to conform with http://roman-ivanov.blogspot.com/2013/10/trailing-comment-check-false-positives.html where it is mentioned that "_**To fix false-positives we can update Trailing Commentary Sonar check to ignore: //NOSONAR, //SUPPRESS CHECKSTYLE, //NOPMD,**_"

Reply:
@AkMo3 , please remove ` OK` from examples. to keep them exactly as it is used in real life.
I know it is a bit against our vision on how examples should looks like, but I think here is exception from rule.

Reply:
Done